.github/workflows/e2e_tests.yml

```
@@ -74,69 +36,47 @@ jobs:
+          echo "Deployment SHA: ${{ github.event.deployment.sha }}"
 
-      # FIXME: This step is not working well. All environments seem to be Preview.
-      - name: Upload test results
-        if: ${{ steps.check.outputs.should_run == 'true' && failure() }}
-        uses: actions/upload-artifact@ea165f8d65b6e75b540449e92b4886f43607fa02 # v4.6.2
+      - name: liam-app-e2e_tests
+        if: (github.event.deployment_status.state == 'success' &&
+          contains(github.event.deployment.environment, 'Production') &&
+          contains(github.event.deployment_status.target_url, 'liam-app-git-main')) ||
+          (!contains(github.event.deployment_status.target_url, 'liam-docs') && !contains(github.event.deployment_status.target_url, 'liam-erd-sample') && !contains(github.event.deployment_status.target_url, 'liam-storybook'))
+        uses: ./.github/actions/reusable-e2e
```

actionlint (1.7.7)

could not parse action metadata in "/home/jailuser/git/.github/actions/reusable-e2e": yaml: line 63: mapping values are not allowed in this context

(action)